README.md

```diff
@@ -37,48 +37,50 @@ Some test suites use [Model-Based Testing](https://msdn.microsoft.com/en-us/libr
 * **SMB1 Server Test Suite**
 * **File Server Family Test Suite**
 
-To be able to build the above test suites, you should not only install [Spec Explorer](https://visualstudiogallery.msdn.microsoft.com/271d0904-f178-4ce9-956b-d9bfa4902745/), but also build [Protocol Test Framework](https://github.com/microsoft/protocoltestframework)
-by **formodel** option and install the built **ProtocolTestFrameworkInstaller.msi**.
-
 ## Build
 
-After you [clone a copy](https://help.github.com/articles/cloning-a-repository/) of this repo, change to your local **WindowsProtocolTestSuites** directory:
-
-```
-cd WindowsProtocolTestSuites
-```
+After you [clone a copy](https://help.github.com/articles/cloning-a-repository/) of this repo, you can run **build.cmd** for Protocol Test Manager and each test suite separately.
 
-Run **buildall.cmd**
+### Build Protocol Test Manager
 
 ```
-buildall.cmd
+cd WindowsProtocolTestSuites\ProtocolTestManager
+build.cmd
 ```
 
-After the build succeeds, the MSI (installer package) file of every test suite should be generated in the folder *WindowsProtocolTestSuite\drop\TestSuites\\[TestSuiteName]\Installer\.*
-Take SMB test suite as an example, **MS-SMB-TestSuite-ServerEP.msi** should be generated in the folder *WindowsProtocolTestSuite\drop\TestSuites\MS-SMB\Installer\.*
+###<a name="BuildTestSuite">Build a test suite
 
-You can also run **build.cmd** for protocol SDK, Protocol Test Manager and every test suite separately.
+If the test suite contain Model-Based Test cases, you should follow the below steps before building it. 
 
-To build protocol SDK
+* Install [Spec Explorer](https://visualstudiogallery.msdn.microsoft.com/271d0904-f178-4ce9-956b-d9bfa4902745/), 
+* Install the **formodel** version of [Protocol Test Framework](https://github.com/microsoft/protocoltestframework). You can use a released MSI file or build it from source code by **formodel** option.
```

Comment:
Add detail for PTF: +  cd src
                               +  build.cmd formodel